### PR TITLE
ISLANDORA-1444 default option says "(default)".

### PR DESCRIPTION
**Pull request by rosiel**

### CODE REVIEW

**Comment by manez** [comment]
Language makes sense to me.

**Comment by DiegoPino** [comment]
:+1:

File: islandora_populator.module [hunk 1]
```diff
@@ -98,3 +98,14 @@ function islandora_populator_islandora_ingest_steps(array &$form_state) {
 
   return $steps;
 }
+
+/**
+ * Impelements hook_islandora_ingest_steps_alter().
```
Comment (adam-vessey) [inline islandora_populator.module]: "Implements" is misspelled.

File: islandora_populator.module [hunk 2]
```diff
@@ -98,3 +98,14 @@ function islandora_populator_islandora_ingest_steps(array &$form_state) {
...
+ */
+function islandora_populator_islandora_ingest_steps_alter(array &$steps, array &$form_state) {
+  if (isset($steps['xml_form_builder_metadata_step'])) {
+    if (array_key_exists('islandora_populator_select', $steps)) {
+      $steps['islandora_populator_select']['args']['0']['default']['description'] = t("An opportunity will be provided to enter metadata.");
```
Comment (adam-vessey) [inline islandora_populator.module]: Shouldn't really be an issue 'cause PHP, but the offset inside `'args'` should be an integer, not a string.

**Comment by rosiel** [comment]
Thanks for the code review @adam-vessey!

**Pull request merged**